lib/widgets/content.dart

```diff
@@ -1196,6 +1235,58 @@ class GlobalTime extends StatelessWidget {
   }
 }
 
+class MessageTable extends StatelessWidget {
+  const MessageTable({super.key, required this.node});
+
+  final TableNode node;
+
+  @override
+  Widget build(BuildContext context) {
+    final contentTheme = ContentTheme.of(context);
+    return SingleChildScrollViewWithScrollbar(
+      scrollDirection: Axis.horizontal,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 5),
+        child: Table(
+          border: TableBorder.all(
+            width: 1,
+            style: BorderStyle.solid,
+            color: contentTheme.colorTableCellBorder),
+          defaultColumnWidth: const IntrinsicColumnWidth(),
+          children: List.unmodifiable(node.rows.map((row) => TableRow(
+            decoration: row.isHeader
+              ? BoxDecoration(color: contentTheme.colorTableHeaderBackground)
+              : null,
+            children: List.unmodifiable(row.cells.map((cell) =>
+              MessageTableCell(node: cell, isHeader: row.isHeader)))))))));
+  }
+}
+
+class MessageTableCell extends StatelessWidget {
+  const MessageTableCell({super.key, required this.node, required this.isHeader});
+
+  final TableCellNode node;
+  final bool isHeader;
+
+  @override
+  Widget build(BuildContext context) {
+    return TableCell(
+      verticalAlignment: TableCellVerticalAlignment.middle,
+      child: Padding(
+        padding: const EdgeInsets.all(4),
```

Comment:
```suggestion
        // Web has 4px padding and 1px border on all sides.
        // In web, the 1px border grows each cell by 0.5px in all directions.
        // Our border doesn't affect the layout, it's just painted on,
        // so we add 0.5px on all sides to match web.
        padding: const EdgeInsets.all(4 + 0.5),
```

Reply:
Interesting, found an upstream issue for this: https://github.com/flutter/flutter/issues/78691.
Included the link for reference.